src/numerical_flux/central_numerical_flux.hpp

```diff
@@ -0,0 +1,41 @@
+#ifndef __CENTRAL_NUM_FLUX__
+#define __CENTRAL_NUM_FLUX__
+
+#include <deal.II/base/tensor.h>
+#include "physics/physics.h"
+#include "numerical_flux/convective_numerical_flux.hpp"
+
+namespace PHiLiP {
+namespace NumericalFlux {
+
+/// Central numerical flux. Derived from NumericalFluxConvective.
+template<int dim, int nstate, typename real>
+class CentralNumFlux: public NumericalFluxConvective<dim, nstate, real>
```

Comment:
consider moving this into `convective_numerical_flux.hpp`

Reply:
fixed as per issue #125